apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -71,7 +76,7 @@ const customProjectProperties: CustomProjectProperties = {
     testIgnore: WAGMI_DISABLED_TESTS_REGEX
   },
   'Desktop Firefox/wagmi': {
-    testIgnore: WAGMI_DISABLED_TESTS_REGEX
+    testIgnore: WAGMI_DISABLED_TESTS_REGEX_FF
```

Comment:
does it not work on FF?

Reply:
Couldn't find any reference to Firefox on Synpress' documents. Need to ask them in Discord